docs/source/user/fast.farm/index.rst

```diff
@@ -1,15 +1,14 @@
-.. _FAST.Farm:
 
-FAST.Farm User’s Guide and Theory Manual
+FAST.Farm User's Guide and Theory Manual
 ========================================
 
-.. only:: html
+This document offers a quick reference guide for the FAST.Farm driver for
```

Comment:
The sentence, "This document offers a quick reference guide for the FAST.Farm driver for OpenFAST." should be deleted.

Reply:
See #1178 for correction